src/algorithms/model.ts

```diff
@@ -0,0 +1,376 @@
+import { AllParamsFlat } from '../algorithms/types/Param.types'
+import { SeverityTableRow } from '../components/Main/Scenario/SeverityTable'
+import { ModelParams, SimulationTimePoint } from '../algorithms/types/Result.types'
+
+const msPerDay = 1000 * 60 * 60 * 24
+
+const monthToDay = (m: number) => {
+  return m * 30 + 15
+}
+
+const jan2020 = new Date('2020-01-01').valueOf() // time in ms
```

Comment:
This was using implicit `Date` -> `number` conversion so I am now explicitly converting to a `number`.